src/components/standings.rs

```diff
@@ -114,30 +125,84 @@ impl StandingsState {
         self.date_selector.set_date_with_arrows(forward)
     }
 
+    /// Toggle between division view and overall view
+    pub fn toggle_view_mode(&mut self) {
+        self.view_mode = match self.view_mode {
+            ViewMode::ByDivision => ViewMode::Overall,
+            ViewMode::Overall => ViewMode::ByDivision,
+        };
+        self.team_ids = self.generate_ids();
+        self.reset_selection();
+    }
+
+    /// Get all teams sorted by record (for overall view)
+    pub fn get_teams_by_record(&self) -> Vec<Standing> {
```

Comment:
basically use this to create a `league_standings` vec once. could also make this non public